src/styles/calendar.ts

```diff
@@ -11,15 +11,15 @@ type DayDivTaskProps = {
   hasEvent: boolean;
 };
 
-export const CalendarContainer = styled.div`
+export const Container = styled.div`
```

Comment:
Could you please change to CalendarContainerStyled  as when imported it can be confused with the <Continaer/> component from MUI ?

Reply:
ok